content/docs/sidebar.json

```
@@ -100,8 +100,8 @@
         "slug": "how-to",
         "source": false,
         "children": [
-          "add-output-to-stage",
           "undo-adding-data",
+          "add-deps-or-outs-to-a-stage",
```

Not sure about the label. Let's try:
```suggestion
        {
          "label": "Add Tracked Data to a Stage",
          "slug": "add-deps-or-outs-to-a-stage"
        },
```